apps/apps.h

```diff
@@ -568,7 +568,7 @@ void store_setup_crl_download(X509_STORE *st);
 
 # define APP_PASS_LEN    1024
 
-# define SERIAL_RAND_BITS        64
+# define SERIAL_RAND_BITS        (18 * 8)
```

Comment:
I guess I prefer to use the maximum length, which would be 20. But there has been some discussion on what that means exactly, so it might be useful to have 159 bits instead. 160 bit with the top bit set would create something that's 21 bytes in the encoding.